lib/build.command.js

```diff
@@ -9,36 +9,35 @@
 module.exports = function (Aquifer) {
   'use strict';
 
+  // If the current directory doesn't contain an aquifer project, exit.
+  if (!Aquifer.initialized) {
+    return;
```

Comment:
On second thought, I'd like to keep this pattern. Command files often add multiple commands to the CLI, some of which should be available to only non-initialized Aquifer instances, and some that should be available only to initialized Aquifer instances. It's especially critical to establish this pattern, considering the upcoming extensions library.